### PR TITLE
Update to Modal and Dialog HTML structure

### PR DESCRIPTION
**Pull request by pauljeter**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/collab-ui/collab-ui-react/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [x] Feature
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

What is the current behavior? (You can also link to an open issue here)
Modals have a left and right page inside of the cui-modal__content div for the dialogs to position the icon.

What is the new behavior?
Dialogs no longer have an icon in the left-pane, so the panes are being removed

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
Dependent on https://github.com/collab-ui/collab-ui-core/pull/155

### CODE REVIEW

File: src/docs/containers/Playground/insertExampleHere.js [hunk 1]
```diff
@@ -1,16 +1,188 @@
...
 
 // Import Method Show Below
-// import { Button } from '@collab-ui/react';
+import {   Button,
+  Modal,
+  ModalBody,
+  ModalFooter,
+  ModalHeader } from '@collab-ui/react';
 
 export default class PlaygroundComponent extends React.Component {
```
Comment (daanishnasir) [inline src/docs/containers/Playground/insertExampleHere.js]: Playground component changes @pauljeter

File: src/docs/containers/Playground/insertExampleHere.js [hunk 2]
```diff
@@ -1,16 +1,188 @@
...
+                  e.preventDefault();
+                  this.modal2.closeModal();
+                }}
+                ariaLabel='Submit Form'
+                color='blue'
+              />
+            </ModalFooter>
+          </Modal>
+
+                    <Button
```
Comment (daanishnasir) [inline src/docs/containers/Playground/insertExampleHere.js]: tab

File: src/docs/containers/Playground/insertExampleHere.js (hunk 1 above)
Comment (pauljeter) [reply src/docs/containers/Playground/insertExampleHere.js]: Whoops!

File: src/docs/containers/Playground/insertExampleHere.js (hunk 2 above)
Comment (pauljeter) [reply src/docs/containers/Playground/insertExampleHere.js]: All of this has been removed.

File: src/lib/Modal/index.js [hunk 3]
```diff
@@ -55,28 +55,15 @@ class Modal extends React.Component {
...
     const modalContent = (
-      <div className={`cui-modal__content`}>
-        {icon && size === 'dialog' && getIcon() && (
-          <div className="cui-modal__content--left-pane">
-            {icon}
-          </div>
-        )}
-        <div className="cui-modal__content--right-pane">
+      <div className="cui-modal__content">
+        <div className={`cui-modal__flex-container`}>
```
[the remark this answers is not in the document]
Comment (pauljeter) [reply src/lib/Modal/index.js]: Fixed.

**Pull request merged**